jetpack.php

```diff
@@ -56,6 +56,37 @@ function jetpack_require_lib_dir() {
 	return JETPACK__PLUGIN_DIR . '_inc/lib';
 }
 
+/**
+ * THIS CODE WOULD NEED TO BE DUPLICATED IN EACH PLUGIN...
```

Comment:
Or it could be a separate package on its own 😉

Reply:
great idea! This was more of a proof of concert.